usaspending_api/awards/tests/unit/test_location_filter_geocode.py

```diff
@@ -208,8 +208,7 @@ def test_validate_location_keys():
                     "zip": "12345",
                     "city": "Chicago",
                     "state": "IL",
-                    "county": "Yes",
```

Comment:
Why did you remove "county"?

Reply:
These tests cover valid inputs and county is not valid with the district_original term. I just added a valid test that contains the county term to make sure we are still covering this scenario of having the county term present in a valid input.